libopenage/testing/CMakeLists.txt

```diff
@@ -6,3 +8,8 @@ add_sources(libopenage
 pxdgen(
 	testlist.h
 )
+
+target_include_directories(libopenage
+	PRIVATE
+		${CMAKE_CURRENT_SOURCE_DIR}
```

Comment:
I don't think we'll need that, we already added the `libopenage` build dir as includedir and should refer relatively then?

Reply:
It could work if I change `testlist.cpp.template` to `#include "testing/testlist.h"`, but that might be disobeying [mom](https://github.com/SFTtech/openage/blob/master/doc/code_style/mom.cpp#L18).
Thoughts?

Reply:
Ah :) I think that's fine, mom won't be mad if we don't write exact minimal header names in generated files.